code/src/components/NewThought.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+
+export const NewThought = ({ onChangeThought }) => {
+    const [newThought, setNewThought] = useState("")
+
+    const handleSubmit = e => {
+        e.preventDefault()
+        onChangeThought(newThought)
+        setNewThought('')
+    }
+
+    const handleMessages = e => {
+        setNewThought(e.target.value)
+    }
+
+    return (
+        <div className="new-thought-section">
+            <form onSubmit={handleSubmit}>
+                <label for="title-thought-section">What's making you happy right now?</label>
```

Comment:
I think here if you want the label to be connected to the textarea then the label's "for" attribute should be the same as the textarea's "id" attribute, that way they are connected. 
Or this could be a <p> element and no "for" is needed.